assets/scripts/util.js

```diff
@@ -332,14 +336,14 @@ function radio_trend(category, measured, target) {
   return CATEGORIES[category][2];
 }
 
-var DIRECTIONS = ["N", "NE", "E", "SE", "S", "SW", "W", "NW"];
+function radio_altitude(alt) {
+  if(alt >= 18000) return "flight level " + parseInt(alt.toString().substr(0,3));
```

Comment:
Yeah you may be right. And now that I think about it, in Europe they can have some flight levels below 10,000ft, eg 7000 being "Flight Level Seven Zero", which would be a different number of digits... Just for the future it should probably be set up as you suggested.

And I'm not sure what you mean about altitudes that aren't rounded by 100... But all that ATC would see is precision to the hundreds-of-ft level, if that's what you're getting at. Expressing tens of feet is not needed either in communications or in display, so I am not worried about that.